doc/wintabs.txt

```diff
@@ -200,6 +200,32 @@ default: ' '
                 '[]' and ')(' are some of the good ones. It's recommended that
                 they have the same length as separators.
 
+*g:wintabs_ui_show_vimtab_name*
+values: 0, 1 or 2
+default: 0
+                This controls how Vim tabs will be displayed.
+                0: show a number of the tab.
+                1: show a name of the tab.
+                2: show number and name of the tab. (<number>:<tab_name>).
```

Comment:
In additional, I can map command `tabn N` to some shortcut and next I'll be able to use number of tabs for faster switching between opened tabs.